tools/cli/internal/cli/split/split_test.go

```diff
@@ -34,6 +38,83 @@ func TestSuccessfulSplit_Run(t *testing.T) {
 	}
 }
 
+func TestSplitPublicPreviewRun(t *testing.T) {
```

Comment:
I use the command locally to generate the spec and I noticed that the private preview file has several components that I think are not needed. Is this expected? 
[test-private-preview-info-resource.json](https://github.com/user-attachments/files/18873133/test-private-preview-info-resource.json)

I am wondering if this is okay and will be fixed once we add the filter to remove the unused view 🤔

Reply:
Yes, we should cover removing unused stuff after split together, for now we're validating we're filtering out the paths, which is the most important resource